tests/unit/ForeignJurisdiction.spec.ts

```diff
@@ -66,7 +66,7 @@ describe('ForeignJurisdiction', () => {
 
     const provinces = wrapper.findAll('.v-select').at(1).props('items')
 
-    expect(!provinces.includes({'name': 'British Columbia', 'short': 'BC'}))
+    expect(provinces).toEqual(expect.not.arrayContaining([{name: 'British Columbia', short: 'BC'}]))
```

Comment:
Thanks for figuring this out. Can it be simplified?
`expect(provinces).not.arrayContaining([{name: 'British Columbia', short: 'BC'}])`

Reply:
This didn't work. Sev, for reference: 
![not containing](https://github.com/bcgov/business-filings-ui/assets/122301442/09a8361b-35d8-4605-8105-6231bc75bc55)

Reply:
I have to stop doubting that you've already tried everything, lol.

Fair enough. This is resolved.